bin/start_stream

```diff
@@ -28,7 +28,8 @@ begin
 
   outputs = [
     Stats::Output::ElasticsearchWriter.new,
-    Stats::Output::PusherWriter.new
+    Stats::Output::PusherWriter.new,
+    Stats::Output::TimescaleWriter.new
```

Comment:
switch the order of these to ensure pusher is last in the list, so it only executes if the ES and PG writers succeed.